analytics-webapps/src/main/webapp/js/statistic-collection.js

```diff
@@ -422,8 +447,6 @@ function() {
                 pageUri: eXo.env.portal.selectedNodeUri,
                 applicationName: appName,
                 isMobile,
-                deviceType: deviceType,
```

Comment:
Should we keep theses lines ?

Reply:
i've refactored the code so that we are sending the data on a single event when the page is loaded